packages/web/server/queries/complex/assets/price.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from "~/server/queries/coingecko";
```

Comment:
What are your throughs on adding a data loader for calculating prices with pools on this PR?

Reply:
You mean a data loader for fetching pools by ID?

Reply:
I was looking into that and, with our use of pool providers, our direct use of this legacy pools query fn, and with the SQS prices API service nearing completion, I thought I'd wait for that then just remove the use of this query.